starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/micronautGradle.rocker.raw

```diff
@@ -1,7 +1,33 @@
 @import java.util.List
 @import io.micronaut.starter.feature.build.gradle.Dockerfile
 @import io.micronaut.starter.options.BuildTool
-@args(BuildTool build, Dockerfile dockerfile, Dockerfile dockerfileNative, List<String> dockerBuilderImages, List<String> dockerBuilderNativeImages)
+@args(BuildTool build, Dockerfile dockerfile, Dockerfile dockerfileNative, List<String> dockerBuilderImages, List<String> dockerBuilderNativeImages, String runtime, String testRuntime, String aotVersion, boolean incremental, String packageName)
+micronaut {
+@if(runtime != null) {
+    runtime("@(runtime)")
+}
+@if(testRuntime != null) {
+    testRuntime("@(testRuntime)")
+}
+@if(incremental) {
+    processing {
+        incremental(true)
+        annotations("@(packageName).*")
+    }
+}
+@if(aotVersion != null) {
+    aot {
+        version = '@(aotVersion)'
```

Comment:
What about the Kotlin DSL? If I understand this template is for both `.gradle` and `.gradle.kts` but this block would fail with .kts.

Reply:
I have added Kotlin DSL support here: https://github.com/micronaut-projects/micronaut-starter/pull/1043/commits/cacc08c1b81444b93128fe1ba94a8d588ea622dd